docs/src/tcp.rst

```diff
@@ -113,3 +113,11 @@ API
         mapping
 
 .. seealso:: The :c:type:`uv_stream_t` API functions also apply.
+
+.. c:function:: int uv_tcp_close_reset(uv_tcp_t* handle, uv_close_cb close_cb)
+
+    It resets a TCP connection by sending a RST packet. Due to some platform
+    inconsistencies, mixing of :c:func:`uv_shutdown` and
+    :c:func:`uv_tcp_close_reset` calls is not allowed.
```

Comment:
I’d make it explicit that this calls `uv_close()`.

Reply:
Docs updated. Thanks!